src/main/java/org/apache/pulsar/manager/PulsarApplicationListener.java

```diff
@@ -59,34 +59,37 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
                 .getEnvironmentsList(1, 1);
         if (environmentEntities.getResult().size() <= 0) {
             Optional<EnvironmentEntity> environmentEntityOptional = environmentsRepository
-                    .findByName(environmentDefaultName);
-            if (environmentDefaultName != null
-                    && environmentDefaultBroker != null
-                    && environmentDefaultName.length() > 0
-                    && environmentDefaultBroker.length() > 0
+                    .findByName(defaultEnvironmentName);
+            if (defaultEnvironmentName != null
+                    && defaultEnvironmentServiceUrl != null
+                    && defaultEnvironmentName.length() > 0
+                    && defaultEnvironmentServiceUrl.length() > 0
                     && !environmentEntityOptional.isPresent()) {
                 Map<String, String> header = Maps.newHashMap();
                 header.put("Content-Type", "application/json");
                 if (StringUtils.isNotBlank(pulsarJwtToken)) {
                     header.put("Authorization", String.format("Bearer %s", pulsarJwtToken));
                 }
-                String httpTestResult = HttpUtil.doGet(environmentDefaultBroker + "/metrics", header);
+                String httpTestResult = HttpUtil.doGet(defaultEnvironmentServiceUrl + "/metrics", header);
                 if (httpTestResult != null) {
                     EnvironmentEntity environmentEntity = new EnvironmentEntity();
-                    environmentEntity.setBroker(environmentDefaultBroker);
-                    environmentEntity.setName(environmentDefaultName);
+                    environmentEntity.setBroker(defaultEnvironmentServiceUrl);
+                    environmentEntity.setName(defaultEnvironmentName);
                     environmentsRepository.save(environmentEntity);
-                    log.info("The default environment set successful");
+                    log.info("Successfully added a default environment: name = {}, service_url = {}.",
+                            defaultEnvironmentName, defaultEnvironmentServiceUrl);
                 } else {
-                    log.error("The default environment configuration is incorrect, please check " +
-                            "configuration environment.default.name and environment.default.broker");
+                    log.error("Unable to connect default environment {} via {}, " +
+                            "please check if `environment.default.name` " +
+                            "and `environment.default.broker` are set correctly, " +
```

Comment:
The log message is incorrect, the property has changed to default.environment.service_url, however log contains old name.